frontend/markup/product/product.css

```diff
@@ -0,0 +1,295 @@
+* {
+  -webkit-transition: .25s ease-in-out;
+  -moz-transition: .25s ease-in-out;
+  -o-transition: .25s ease-in-out;
+  transition: .25s ease-in-out;
+}
+body {
+  margin: 0px;
+  background-color: #ffffff;
+  position: relative;
+}
+
+body * {
+  box-sizing: border-box;
+}
+
+button {
+  box-shadow: none;
+  outline: none;
+  border: none;
+  cursor: pointer;
+}
+
+.product-shield {
+	position: fixed;
+	top: 0;
+	left: 0;
+	right: 0;
+	bottom: 0;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+}
+
+.product-splash {
+  width: 715px;
+  height: 456px;
+  border-radius: 4px;
+  background-color: #ffffff;
+  box-shadow: 0 5px 7px 0 rgba(0, 0, 0, 0.3);
+  position: relative;
+}
+
+.product-splash__gallery {
+	overflow: hidden;
+	display: inline-flex;
+	justify-content: center;
+	align-items: center;
+	flex-direction: column;
+	width: 290px;
+	height: 100%;
+}
+
+.product-splash__gallery__image {
+	flex: 1 0 auto;
+}
+
+.product-splash__content {
+	display: inline-flex;
+	width: calc(100% - 294px);
+	height: 100%;
+	vertical-align: top;
+	flex-direction: column;
+	justify-content: space-between;
+	align-items: flex-end;
+}
+
+.product-splash__content::after {
+	position: absolute;
+	content: 'X';
+	width: 9px;
+	height: 9px;
+	right: 20px;
+	top: 16px;
+}
+
+.product-splash__content__main {
+	width: 100%;
+	padding-left: 20px;
+}
+
+.product-splash__content__name {
+	overflow: none;
```

Comment:
'none' seems not a valid value for overflow: https://developer.mozilla.org/en-US/docs/Web/CSS/overflow